app/controllers/admin/dashboard_controller.rb

```diff
@@ -34,6 +35,26 @@ def prep_projects_grid
     @status_filter_options = STATUS_FILTERS.map { |f| [I18n.t("dashboard.status_options.#{f}"), f] }
   end
 
+  # Prepare grids for all users inside selected division
+  def prep_projects_grids
+    @people = @division.people.where(has_system_access: true).where.not(id: @person.id)
+
+    @people_with_projects = []
+    @grids = {}
+    @people.each do |person|
+      if person.agent_projects.length > 0
+        @people_with_projects << person
+        @grids["person_#{person.id}"] = initialize_grid(
```

Comment:
It's unfortunate that this is going to fire N queries where N is the number of users being loaded. Generally we try to avoid this (it's called the "N+1 queries" problem). But I don't see an easy way around it here.